### ISSUE TITLE
Fat Mach-O support

### ISSUE THREAD

**Issue by Absolucy** [namazso/IDAShell#1]
The shell extension doesn't show up for fat (multi-architecture) Mach-O files. I understand it may be a bit... weird, because a fat mach-o file can have both 64 and 32 bit slices, but maybe just default to 64-bit it's present, otherwise 32-bit?

The first "magic" of a fat mach-o file is `0xbebafeca` (little endian), and the slices are just regular Mach-O.

**Comment by namazso** [comment]
hey, i'm about done with this but which ida is needed for things with this set:
```c
#define CPU_ARCH_ABI64_32       0x02000000      /* ABI for 64-bit hardware with 32-bit types; LP32 */
#define CPU_TYPE_ARM64_32       (CPU_TYPE_ARM | CPU_ARCH_ABI64_32)
```
i never used a mac and have no idea where these exist

**Comment by Absolucy** [comment]
According to [this changelog](https://hex-rays.com/products/ida/news/7_2/) (`+ MACHO: support ARM64_32. only in ida64 (for now)`), ida64

i believe arm64_32 binaries are related to watchOS

**Comment by namazso** [comment]
fixed in a2019ab9b1484256525b0a985c267696e5942c9c / v1.0.2
